packages/amplication-server/src/core/auth/auth.service.ts

```diff
@@ -568,7 +620,6 @@ export class AuthService {
     if (apiToken.count === 1) {
       return true;
     }
-    return false;
```

Comment:
Don't you need to return false here (there is no return in this flow, unless you mean an implicit false?

Reply:
Nice one! this change was not on purpose